allauth/socialaccount/providers/eventbrite/provider.py

```diff
@@ -15,6 +15,12 @@ def get_avatar_url(self):
         """Return avatar url."""
         return self.account.extra_data["image_id"]
 
+    def to_str(self):
+        for email in self.account.extra_data.get("emails", []):
+            if email.get("verified") and email.get("email"):
```

Comment:
For display purposes I think it's fine to fallback to a non-verified email if no verified email exists.